src/components/mapgl/mapgl.component.ts

```diff
@@ -495,9 +504,18 @@ export class MapglComponent implements OnInit, AfterViewInit, OnChanges {
       const bounds = (<mapboxgl.Map>this.map).getBounds();
       const mapExtend: MapExtend = { bounds: bounds.toArray(), center: bounds.getCenter().toArray(), zoom: this.map.getZoom() };
       this.onMapClosed.next(mapExtend);
-    }
-    );
+    });
 
+    this.finishDrawTooltip = document.getElementById('polygon-finish-draw-tooltip');
+    this.validateDrawTooltip = document.getElementById('polygon-validate-draw-tooltip');
+    fromEvent<MouseEvent>(window, 'mousemove').subscribe((e) => {
+      const x = e.clientX;
+      const y = e.clientY;
+      this.finishDrawTooltip.style.top = (y + 20) + 'px';
+      this.finishDrawTooltip.style.left = (x + 20) + 'px';
+      this.validateDrawTooltip.style.top = (y + 20) + 'px';
+      this.validateDrawTooltip.style.left = (x + 20) + 'px';
+    });
```

Comment:
This event mustn't be listen permanently, only when the user draw a polygon or update an existing one

Reply:
fixed